naxsi_src/naxsi_skeleton.c

```diff
@@ -247,18 +270,96 @@ static ngx_command_t  ngx_http_dummy_commands[] =  {
 };
 
 /*
+ * Variables
+ */
+static ngx_http_variable_t ngx_http_naxsi_variables[] = {
```

Comment:
What about macroifying this to avoid probable/possible mistakes due to copy-pasta?